### PR TITLE
FIX: reload was mad slowed down. it's good now

### PR DESCRIPTION
**Pull request by chipsams**
ever-increasing reload times starting at 35 seconds to a nice clean 3.5s, fresh!

### CODE REVIEW

File: kubejs/server_scripts/base/process.js [hunk 1]
```diff
@@ -23,25 +24,34 @@ ServerEvents.recipes(event => {
...
-            if ((recipe.output + "").includes(item+"")) {
-                
-                r = true
-            }
-        }
-    });
-    return r
+  if(!lazy_removedRecipesMap) lazy_removedRecipesMap = createRemovedRecipeMap(global.itemsToRemove);
+
+  let r = false;
```
Comment (EngineerSmith) [inline kubejs/server_scripts/base/process.js:42]: This variable isn't used anymore

Comment (chipsams) [reply kubejs/server_scripts/base/process.js:42]: thanks!

File: kubejs/startup_scripts/blocks.js [hunk 2]
```diff
@@ -41,10 +41,27 @@ StartupEvents.registry("item", (event) => {
...
+      },
+      "axis=y": {
+        "model": "ptdye:block/orange_tube_block"
+      },
+      "axis=z": {
+        "model": "ptdye:block/orange_tube_block",
+        "x": 90
+      }
+    }
+  }
```
Comment (chipsams) [inline kubejs/startup_scripts/blocks.js:64]: i'm guessing adding a new block is fine? shouldn't collide with anything...

**Review by chipsams — COMMENTED** [review]
ok, yayy! it doesn't change anything about gameplay now i think? was the raw white -> plates change a thing before. it doesn't matter probably...

File: kubejs/startup_scripts/blocks.js (hunk 2 above)
Comment (Yarden-zamir) [reply kubejs/startup_scripts/blocks.js:64]: Yes, adding a block is fine

**Review by Yarden-zamir — APPROVED** [review]

**Pull request merged**